src/game/Objects/Pet.h

```diff
@@ -198,9 +198,12 @@ class Pet : public Creature
         int32 GetBonusDamage() const { return m_bonusdamage; }
         void SetBonusDamage(int32 damage) { m_bonusdamage = damage; }
 
+        bool UpdateStats(Stats stat) override;
```

Comment:
Why does it show these functions have been re-added to Pet class?

Reply:
Actually the whole CLS implementation commit seems to be reverted by your pull request :O

https://github.com/vmangos/core/commit/358ab70d8892a9bf9a4cc3f0c57294b9338ce9c7

Reply:
This is probably a merge error when I rebased my branch off the current development branch.  Ugh.  I guess I'll have to go through the whole thing manually.

Reply:
This should be fixed now.